python/lsst/pipe/tasks/processCcdWithFakes.py

```diff
@@ -237,12 +247,15 @@ def runDataRef(self, dataRef):
             photoCalib = calexp.getPhotoCalib()
 
         icSourceCat = dataRef.get("icSrc", immediate=True)
+        sfdSourceCat = dataRef.get("src", immediate=True)
 
         resultStruct = self.run(fakeCat, calexp, wcs=wcs, photoCalib=photoCalib,
-                                exposureIdInfo=exposureIdInfo, icSourceCat=icSourceCat)
+                                exposureIdInfo=exposureIdInfo, icSourceCat=icSourceCat,
+                                sfdSourceCat=sfdSourceCat)
```

Comment:
sfdSourceCat is not listed in the Parameters section of the docstring.  Neither is icSourceCat, which isn't technically in scope here, but should be documented here as well (so N_parameters = N_documentedParameters).  If you could state what an "sfd" is as well, so it's clear what the content should be ("single frame driver"?).

Reply:
Thank you for catching both of these.